client/src/routes/fight/FightSequence.svelte

```diff
@@ -64,6 +64,9 @@
 <style>
     div {
         box-sizing: border-box;
-        height: 48vh;
+        height: 27.5vh;
+        border-bottom: 2px solid #4161FE;
+        box-sizing: border-box;
+        padding: 1vh 0;
```

Comment:
Nie używałbym jednostki `vh` do paddingu. To jest dobre miejsce na użycie jednostski `em` albo `px`.

Reply:
Popieram skorzystanie z `em` do paddingu

Reply:
Zgadzam się, do paddingu lepiej używać jednostek `em`